big_tests/tests/amp_big_SUITE.erl

```diff
@@ -305,6 +331,30 @@ notify_deliver_to_online_user_recipient_privacy_test(Config) ->
               client_receives_nothing(Bob)
       end).
 
+notify_deliver_to_online_user_broken_connection_test(Config) ->
+    escalus:fresh_story(
+      Config, [{alice, 1}, {bob, 1}],
+      fun(Alice, Bob) ->
+              %% given
+              Rule = {deliver, case ?config(offline_storage, Config) of
+                                   mam -> stored;
+                                   _ -> none
+                               end, notify},
+              Rules = rules(Config, [Rule]),
+              Msg = amp_message_to(Bob, Rules, <<"Recipient connection breaks">>),
```

Comment:
As far as I can see there is a mock set up on the server expecting exactly this message. I'm not sure if we can cleanly make this message configurable in both places so that we don't introduce a strange issue if someone changes the message in only one place in the future. The least we can do is to add a line of comment saying about the mock running on the server and returning an error from the socket if this exact message is observed.